odoo/addons/base/migrations/12.0.1.3/end-migration.py

```diff
@@ -104,11 +104,11 @@ def fork_off_system_user(env):
         "UPDATE res_users " + set_query + " WHERE id = %s",
         (user_root.id, ),
     )
-    # Ensure also partner_root is inactive
+    # Ensure also partner_root and partner_admin are inactive
     env.cr.execute(
         """ UPDATE res_partner
-            SET active = FALSE WHERE id = %s """,
-        (partner_root.id,))
+            SET active = FALSE WHERE id IN %s """,
+        ((partner_root.id, partner_admin.id),))
```

Comment:
I think setting the admin user is incorrect. On new Odoo 12 databases there is an active admin user.

Reply:
Oh I see. It's weird that's not just demo data. I guess this could have been a side effect of seeing a warning in the admin user form, but yes probably this isn't the real fix to the problem.

I'll revert this.